postgres/assets/configuration/spec.yaml

```diff
@@ -86,6 +86,43 @@ files:
         type: string
         display_default: false
         example: "false"
+    - name: ssl_root_cert
+      description: |
+        The path to the ssl root certificate.
+
+        For a detailed description of how this option works see https://www.postgresql.org/docs/current/libpq-ssl.html
```

Comment:
The link provided does not describes the possible values. Is the `system` value supported as described here? https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLROOTCERT